app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -512,6 +458,85 @@ public void sectionsLoaded(String title, List<DocumentSection> sections) {
     }
   }
 
+  private void setUpTabButtons() {
+    newTabButton.setOnClickListener((View view) -> newTab());
+    tabForwardButtonContainer.setOnClickListener((View view) -> {
+      if (getCurrentWebView().canGoForward()) {
+        getCurrentWebView().goForward();
+      }
+    });
+    tabBackButtonContainer.setOnClickListener((View view) -> {
+      if (getCurrentWebView().canGoBack()) {
+        getCurrentWebView().goBack();
+      }
+    });
+  }
+
+  private void setUpTableDrawerAdapter() {
+    tableDrawerAdapter = new TableDrawerAdapter();
+    tableDrawerRight.setAdapter(tableDrawerAdapter);
+    tableDrawerAdapter.setTableClickListener(new TableClickListener() {
```

Comment:
Can you please check if this anonymous listener can be converted to lambda or not ? If it can, then please do it.